src/resolvers/dns.browser.js

```diff
@@ -0,0 +1,3 @@
+'use strict'
+
+module.exports = require('dns-over-http-resolver')
```

Comment:
I think this setup might be problematic for electron (it's not currently being run in CI). This will likely need to account for the browser vs Node.js check in the same file.